core/unit_test/neighbor_unit_test.hpp

```diff
@@ -77,12 +77,14 @@ createParticles( const int num_particle, const double box_min,
                  const double box_max )
 {
     using DataTypes = Cabana::MemberTypes<double[3]>;
-    using AoSoA_t = Cabana::AoSoA<DataTypes, TEST_MEMSPACE>;
-    AoSoA_t aosoa( "aosoa", num_particle );
-
-    auto position = Cabana::slice<0>( aosoa );
-    using PoolType = Kokkos::Random_XorShift64_Pool<TEST_EXECSPACE>;
-    using RandomType = Kokkos::Random_XorShift64<TEST_EXECSPACE>;
+    using AoSoA_t_h = Cabana::AoSoA<DataTypes, Kokkos::HostSpace>;
+    AoSoA_t_h aosoa_host( "host_aosoa", num_particle );
+
+    auto position = Cabana::slice<0>( aosoa_host );
+    using PoolType =
+        Kokkos::Random_XorShift64_Pool<Kokkos::DefaultHostExecutionSpace>;
+    using RandomType =
+        Kokkos::Random_XorShift64<Kokkos::DefaultHostExecutionSpace>;
```

Comment:
I would use a preprocessor directive because I assuming we want it back on the device once it is supported?
I don't feel strongly about this.

Reply:
Done - I assume we'll have to do this for SYCL as well